src/Utils.elm

```diff
@@ -0,0 +1,64 @@
+module Utils exposing (timeAgo)
+
+import Time exposing (Time)
+
+
+timeAgo : Time -> Time -> String
+timeAgo time now =
+    let
+        seconds =
+            round ((now - time) / 1000)
```

Comment:
Let's work with floats everywhere and round numbers just before rendering them as strings.